pyrene/src/utils/DateUtils.ts

```diff
@@ -0,0 +1,264 @@
+import {
```

Comment:
[Performance](https://stackoverflow.com/questions/38171465/modules-import-and-destructuring-performances) on import.
Doing so is more performant :

```
import  isExists from 'date-fns/isExists';
import  sub from 'date-fns/sub';
import  add from 'date-fns/add';
import  Duration from 'date-fns/Duration';
import  format from 'date-fns/format';
```

Reply:
✅